src/backend/joanie/core/models/accounts.py

```diff
@@ -41,6 +41,11 @@ class User(BaseModel, auth_models.AbstractUser):
         null=True,
     )
 
+    has_subscribed_to_newsletter = models.BooleanField(
+        verbose_name=_("has subscribed to newsletter"),
+        default=False,
+    )
```

Comment:
This is too generic. We will probably have other newsletters later. IMO you have to specify the `commercial` newsletter.
Also, a default non commercial newsletter is also used, if we want to add a panel with newsletter management, this one should also be managed.

IMO a dedicated "module" for newsletter should be better.